### PR TITLE
Add ability to pass host path in base and drive methods

### PR DESCRIPTION
**Pull request by rabingaire**

### CODE REVIEW

**Review by xeust — APPROVED** [review]

File: src/constants/url.ts [hunk 1]
```diff
@@ -1,7 +1,33 @@
-const base = process.env.DETA_BASE_HOST?.trim() || 'database.deta.sh/v1';
-const drive = process.env.DETA_DRIVE_HOST?.trim() || 'drive.deta.sh/v1';
+const url = {
+  BASE: `https://:host/:project_id/:base_name`,
```
Comment (aavshr) [inline src/constants/url.ts:2]: can we please append `v1` here?

File: src/constants/url.ts [hunk 2]
```diff
@@ -1,7 +1,33 @@
...
+
+/**
+ * base function returns API URL for base
+ *
+ * @param {string} [host]
+ * @returns {string}
+ */
+function base(host?: string): string {
+  const hostPath =
+    host?.trim() || process.env.DETA_BASE_HOST?.trim() || 'database.deta.sh/v1';
```
Comment (aavshr) [inline src/constants/url.ts:14]: also here can we just have `database.deta.sh`

**Review by aavshr — CHANGES_REQUESTED** [review]
thanks @rabingaire , left some comments

**Review by aavshr — APPROVED** [review]
thanks @rabingaire

**Pull request merged**